Slim/Plugin/InternetRadio/strings.txt

```diff
@@ -39,7 +41,8 @@ PLUGIN_RADIOTIME_MODULE_NAME_DESC
 	ES	TuneIn Radio - Radio por Internet local
 	FI	TuneIn Radio - paikallinen internetradio
 	FR	TuneIn Radio - Radios Internet
-	IT	TuneIn Radio - Radio Internet locale
+	HU	TuneIn Radio - Radios Internet
+	IT	TuneIn Radio - Radio Internet locale	
```

Comment:
Please try to avoid all trailing whitespace (there's a tab after the Italian string).

Reply:
Corrected